### ISSUE TITLE
Change the format of the commandline window?

### ISSUE THREAD

**Issue by zohozer** [develsoftware/GMinerRelease#583]
In the new builds of the Gminer, I have too many informations printed on the console. It is any way how to create a configuration file and select only some elements that are displayed on the output console?

Best regards.

**Comment by k1gg1** [comment]
right-click on the command prompt open -> properties then change font and size from there
